docs/housing-counselor-data.md

```diff
@@ -0,0 +1,163 @@
+# Housing Counselor Data Processing
+
+The [Find a Housing Counselor tool](https://www.consumerfinance.gov/find-a-housing-counselor/) gets its data from the US Department of Housing and Urban Development. HUD provides an API to their list of approved counseling agencies.
```

Comment:
Could we add a link to [the HUD API](https://data.hud.gov/housing_counseling.html) here?